docs/Release_Notes.md

```diff
@@ -67,17 +68,17 @@ JOBS CHANGES
 PARM/CONFIG CHANGES
 -------------------
 
-* No changes from GFS v16.3.21
+* `config.resources.emc.dyn` and `config.resources.nco.static` are changed to reduce the resources accordingly.
```

Comment:
Reduce the resources of what? Since this package includes multiple things now, please specify what job resources are changing.